.gitignore

```diff
@@ -6,5 +6,5 @@
 /.graphql-generated/
 /public/_graphql/
 /data/
-/static/
+/static/[0-9]
```

Comment:
```suggestion
```
There is no `/static/` anymore - just the one in public which you've given its own `.gitignore` file.

Reply:
Updated to /public/static/[0-9]